amqp/1.6/modules/ROOT/pages/amqp-consume.adoc

```diff
@@ -21,6 +21,55 @@ This operation consumes the first available message in the queue identified by t
 After it is received, the message is immediately acknowledged by default. If you want to control the ACK of the message after some processing, then set `ackMode` to `MANUAL`.
 For more information about message ACK operations, see xref:amqp-ack.adoc[How to Handle Acknowledgment].
 
+== Default Configuration for the Consume Operation
+
+The default AMQP global configuration element `<amqp:config>` does not have the suitable parameters for the Consume Operation since the configuration is optimized for the AMQP Listener source.
+
+In some environments, using the default AMQP global configuration with the Consume Operation can return the following error:
+
+[source,xml,linenums]
+----
+com.mule.extensions.amqp.internal.connection.provider.GenericConnectionProvider.initialise:221 @415c7a8a ERROR
+Consumer com.mule.extensions.amqp.internal.client.SingleMessageQueueingConsumer@741fd13b (amq.ctag-BsDMf9v86wa9v3e_mo1p8g) method handleDelivery for channel AMQChannel(amqp://xxxx@xxxx/xxxx,1) threw an exception for channel AMQChannel(amqp://xxxx@xxxx:5672/xxxx,1)
+com.rabbitmq.client.AlreadyClosedException: channel is already closed due to clean channel shutdown; protocol method: #method<channel.close>(reply-code=200, reply-text=OK, class-id=0, method-id=0)
+at com.rabbitmq.client.impl.AMQChannel.ensureIsOpen(AMQChannel.java:258)
+at com.rabbitmq.client.impl.AMQChannel.transmit(AMQChannel.java:427)
+at com.rabbitmq.client.impl.AMQChannel.transmit(AMQChannel.java:421)
+at com.rabbitmq.client.impl.recovery.RecoveryAwareChannelN.basicReject(RecoveryAwareChannelN.java:114)
+at com.rabbitmq.client.impl.recovery.AutorecoveringChannel.basicReject(AutorecoveringChannel.java:438)
+at com.mule.extensions.amqp.internal.connection.channel.MuleAmqpChannel.basicReject(MuleAmqpChannel.java:383)
+at com.mule.extensions.amqp.internal.client.SingleMessageQueueingConsumer.handleDelivery(SingleMessageQueueingConsumer.java:47)
+at com.rabbitmq.client.impl.ConsumerDispatcher$5.run(ConsumerDispatcher.java:149)
+at com.rabbitmq.client.impl.ConsumerWorkService$WorkPoolRunnable.run(ConsumerWorkService.java:104)
+at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
+at java.util.concurrent.FutureTask.run(FutureTask.java:266)
+at org.mule.service.scheduler.internal.AbstractRunnableFutureDecorator.doRun(AbstractRunnableFutureDecorator.java:111)
+at org.mule.service.scheduler.internal.RunnableFutureDecorator.run(RunnableFutureDecorator.java:54)
+at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
+at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
+at java.lang.Thread.run(Thread.java:748)
+----
+
+Usually, this occurs because the broker sends all the messages in the queue to the requester but the Consume Operation processes only the first message and discards the remaining ones. To avoid this behavior, you can change one of the following parameters or both depending your use case:
```

Comment:
Let me know if this is a helpful change to these sentences. Think there was a missing word:
Usually, this occurs because the broker sends all the messages in the queue to the requester while the Consume Operation processes only the first message and discards the remaining ones. To avoid this behavior, change one or both of the following parameters depending upon your use case: